requirements/requirements-py-3.10.txt

```diff
@@ -199,7 +199,6 @@ tomli==2.0.1
 tornado==6.3.2
 tqdm==4.65.0
 traitlets==5.9.0
-trogon==0.5.0
```

Comment:
`trogon` is installed via `tui` extra, but that is not mentioned in the header of this file so I don't think it should be here. Note that it's not there for the py-3.12 which was probably generated fresh. Having it here also trips the `verdi devel check-undesired-imports` since it brings `asyncio` module to verdi.